src/app/components/Blog.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Post from './Post';
+
+export default class Blog extends React.Component {
+  render () {
+    const items = this.props.items.map ((items, index) => { 
```

Comment:
```suggestion
    const items = this.props.items.map ((item, index) => { 
```
лучше item, так как мы работаем с одним элементом. Просто для читабельности